pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
```

Comment:
What do you mean with implicitly imported?

The difference with the other modules where we use this pattern, is that, eg in `pandas.io`, `pandas.io.api` are those functions that are imported top-level (imported in `pandas/__init__.py`), not what makes up the `pandas.io` namespace itself. Here, we don't have a collection of functions (put in `api.py`) which we want to have top level, but we just have the `pandas.io` namespace. So IMO it makes more sense and is cleaner to have it just in the init file here.